src/poetry/console/commands/export.py

```diff
@@ -57,12 +57,11 @@ def handle(self) -> None:
             self.call("lock", " ".join(options))
 
         if not locker.is_fresh():
-            self.line_error(
+            self._io.write_line(
```

Comment:
Probably, `line_error()` is correct here, because warnings should go to stderr instead of stdout. I know, in `installer.py` the warning is written to stdout... Maybe, something that can be harmonized for all warnings in a separate PR...

Reply:
Updated